.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "third_party/grpc-web"]
```

Comment:
We need grpc-web submodule for web streaming.
Not sure when gflags was added.

Reply:
Not intended to make changes. Added back.